Addiding v1 of the bs2EMu workflow

ats2008 opened the pull request.

ats2008 committed:
Addiding v1 of the bs2EMu workflow

ats2008 commented:
### Note :
  - Vertexing needs to be sorted out
    - current issue is that the track from the electrons are sometimes NULL
       -  Need to keep checks

rmanzoni commented:
Hi Aravind,
sorry for eavesdropping and peeking into your work (blame GitHub for notifying me!), but here I have an example of vertex fits with mixed mu-ele pairs
https://github.com/rmanzoni/HNL/blob/master/python/utils/utils.py#L153-L155
hope it helps!

ats2008 committed:
Adding the el-mu vtx fit . thx @rmanzoni

ats2008 commented:
@rmanzoni , Thanks I have added your snippet for e-mu vertexing !

ats2008 committed:
bug fix for branch ordering

ats2008 committed:
Adding the first variable for isolation :  muon trackIsolation  , using the existing c++ implementaion

How to add more variables ?
    1 ) In python
        * define it in python file and import it
    2 )  If u already have an implementation in c++
        * Add it as a member funtion to the Utilities class
        * call it from the python script
